SignalServiceKit/src/Messages/OWSMessageReceiver.m

```diff
@@ -13,9 +13,11 @@
 #import "TSStorageManager.h"
 #import "TSYapDatabaseObject.h"
 #import "Threading.h"
+#import <YapDatabase/YapDatabaseAutoView.h>
 #import <YapDatabase/YapDatabaseConnection.h>
 #import <YapDatabase/YapDatabaseTransaction.h>
 #import <YapDatabase/YapDatabaseView.h>
```

Comment:
nit: I think the `YapDatabaseView.h` imports can go away from most (all?) files.

Reply:
Done.